x/incentives/keeper/gauge_test.go

```diff
@@ -280,33 +114,92 @@ func (suite *KeeperTestSuite) TestGaugesByDenom() {
 		err = suite.App.IncentivesKeeper.MoveUpcomingGaugeToActiveGauge(suite.Ctx, *gauge)
 		suite.Require().NoError(err)
 
```

Comment:
```suggestion
		gauges = suite.App.IncentivesKeeper.GetActiveGauges(suite.Ctx)
		suite.Require().Len(gauges, 1)
```

We want to make sure the gauge was successfully moved to the set of active gauges. This also serves as our active gauge check for non-perpetual gauges, since the current implementation only has one for perpetual ones (line 154)